### ISSUE TITLE
Design: Menu buttons

### ISSUE THREAD

**Issue by GewoonMaarten** [nl-design-system/denhaag#70]
The Menu Button component needs to be styled according to the [Figma design](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System-Gemeente-Den-Haag?node-id=1923%3A4663).

As a base for this we will use the Material-UI `Link` and `PopOver` components instead of the Material-UI `Menu` component.

This will result in two components, one `Menu` component in which multiple `Menu Button` components can exist.

## Acceptance criteria
**Variants implemented**
- [x] With options
- [x] Without options

**States implemented**
- [x] Normal
- [x] Hover
- [x] Focus
- [x] Active

**Animations**
Animations are out of scope at this point.

**Dropdown**
As the dropdown design is not yet complete the dropdown styling will be performed in another issue once the design is finished.

**Requirements**
- [x] Without options variant is represented by a HTML `<a>` component
- [x] The dropdown version is represented by a HTML `<button>` component
- [x] [Alignment](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System-Gemeente-Den-Haag?node-id=1923%3A4663)  requirements are met
- [x] Classes use the BEM naming convention
- [x] Design tokens are identified > proposed > implemented
- [x] Only the `props` and possible values for these `props` which are necessary in our use-case are exposed. All other Material-UI and standard HTML props which are not deemed necessary in our use-case should not be allowed/forwarded to the underlying component.
- [x] Storybook shows HTML code snippet (with classes and accessibility suggestions) (Depends on #267 )

**Comment by mjcarsjens** [comment]
Awaiting Menu subitems design

**Comment by mjcarsjens** [comment]
For inspiration: https://pelacase.com
